src/parameters/all_parameters.cpp

```diff
@@ -112,17 +112,33 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
 
     prm.declare_entry("flux_reconstruction", "cDG",
                       dealii::Patterns::Selection(
-                      "cDG | cSD | cHU | cNegative | cNegative2 | cPlus | c10Thousand | cHULumped"),
+                      "cDG | cSD | cHU | cNegative | cNegative2 | cPlus | c10Thousand | cHULumped | user_specified_value"),
                       "Flux Reconstruction. "
                       "Choices are "
-                      " <cDG | cSD | cHU | cNegative | cNegative2 | cPlus | c10Thousand | cHULumped>.");
+                      " <cDG | cSD | cHU | cNegative | cNegative2 | cPlus | c10Thousand | cHULumped | user_specified_value>.");
+
+    prm.declare_entry("FR_user_specified_correction_parameter_value", "0.0",
+                      dealii::Patterns::Double(-dealii::Patterns::Double::max_double_value, dealii::Patterns::Double::max_double_value),
+                      "User specified flux recontruction correction parameter value. Default value is 0.0. ");
 
     prm.declare_entry("flux_reconstruction_aux", "kDG",
                       dealii::Patterns::Selection(
                       "kDG | kSD | kHU | kNegative | kNegative2 | kPlus | k10Thousand"),
                       "Flux Reconstruction for Auxiliary Equation. "
                       "Choices are <kDG | kSD | kHU | kNegative | kNegative2 | kPlus | k10Thousand>.");
 
+    prm.declare_entry("number_ESFR_parameter_values", "0",
+                      dealii::Patterns::Integer(),
+                      "Number of tested ESFR parameter values");
+
+    prm.declare_entry("ESFR_parameter_values_start", "1e-3",
+                      dealii::Patterns::Double(0.0, dealii::Patterns::Double::max_double_value),
+                      "Minimum ESFR parameter values >0 since logspace vector");
+
+    prm.declare_entry("ESFR_parameter_values_end", "1e-3",
+                      dealii::Patterns::Double(0.0, dealii::Patterns::Double::max_double_value),
+                      "Maximum ESFR parameter values >0 since logspace vector");
+
```

Comment:
are test specific to a test of some kind?

Reply:
it is to be able to run a test over a range of c parameter